querydsl-sql/pom.xml

```diff
@@ -259,6 +259,7 @@
                 <value>target/derby.log</value>
             </property>
           </systemProperties>
+          <argLine>-Duser.timezone=UTC</argLine>
```

Comment:
Just thought of something.

The JaCoCo plugin needs to update the `argLine` before it's passed in.
Could you add the `argLine` property in the `<properties />` section?
It can look like this:
```xml
<properties>
  ...
  <argLine>${argLine} -Duser.timezone=UTC</argLine>
  ...
</properties>
```